views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -181,11 +174,17 @@ define([
             init: function init() {
                 var self = this;
 
+                provider = (config.isCompressed)
+                    ? mediaRecorderProvider(config)
+                    : webAudioProvider(config, assetManager);
+
+                this.initAudioContext();
+
                 return navigator.mediaDevices.getUserMedia({ audio: true })
                     .then(function(stream) {
-                        provider = (config.isCompressed)
-                            ? mediaRecorderProvider(config)
-                            : webAudioProvider(config, assetManager);
+                        if (audioContext && (audioContext.state !== 'running')) {
```

Comment:
Just a small thing, but I would prefer to use predefined constants instead of strings as it's used on multiple places (here and L222). What do you think?

For example:
```
var audioContextStates = {
  SUSPENDED: 'suspended',
  RUNNING: 'running',
  CLOSED: 'closed',
};
```